dev/com.ibm.ws.request.probe.audit.servlet/src/com/ibm/ws/request/probe/audit/servlet/AuditPERestService.java

```diff
@@ -0,0 +1,14 @@
+package com.ibm.ws.request.probe.audit.servlet;
+import com.ibm.wsspi.kernel.service.utils.AtomicServiceReference;
+import com.ibm.wsspi.security.audit.AuditService;
+
+public interface AuditPERestService{
+
+    String AUDIT_PE_REST_SOURCE_NAME = "audit_pe_rest";
+    String AUDIT_PE_REST_FILE_HANDLER_NAME = "AuditPERESTFileHandler";
+
+    void auditEventRESTAuthz(AtomicServiceReference<AuditService> auditServiceRef, Object[] methodParams);
+    void auditEventMemberMgmt01(AtomicServiceReference<AuditService> auditServiceRef, Object[] methodParams);
```

Comment:
```suggestion
    void auditEventRESTAuthz(AuditService auditService, Object[] methodParams);
    void auditEventMemberMgmt01(AuditService auditService, Object[] methodParams);
```

I suggest getting the AuditService from the AtomicServiceReference and if it is not null then to call the method, but otherwise don't call the method.  Obviously this will change the logic some, but it is much cleaner.